cmd/wrap_test.go

```diff
@@ -1,12 +1,80 @@
 package cmd
 
 import (
+	"bytes"
+	"io/ioutil"
 	"testing"
 
 	. "github.com/smartystreets/goconvey/convey"
+	"github.com/spf13/viper"
 )
 
 func TestWrap(t *testing.T) {
-	// Any tests we can get?
-	Convey("wrap", t, nil)
+	Convey("wrap", t, func() {
+		viper.Reset()
+		viper.Set("kubeconfigfiles.directory", "/tmp/")
+
+		Convey("returns error when no -e flag is set", func() {
+			viper.Set("environments", map[string]interface{}{
+				"edge-cmh": "kubeconfig-truss-nonprod-cmh",
+			})
+
+			cmd := rootCmd
+			viper.BindPFlag("TRUSS_ENV", cmd.PersistentFlags().Lookup("env"))
+			buff := bytes.NewBufferString("")
+			cmd.SetOut(buff)
+			cmd.SetArgs([]string{
+				"wrap",
+				"-e",
+				"", // Simulating passing in no flag, here. I think persistent flag state is leaking between tests. TODO: Figure out a better integration testing strategy
```

Comment:
This is my one complaint about Cobra right now - everything dangles off this one global `rootCmd` var, which makes it really hard to spin up isolated instances that can be tested in isolation.